pandas_gbq/gbq.py

```diff
@@ -864,6 +877,8 @@ def read_gbq(
         or
         :func:`google.oauth2.service_account.Credentials.from_service_account_file`
         instead.
+    # TODO: add parameters for auth_redirect_uri, client_id, client_secret
```

Comment:
Need to complete this TODO

Reply:
Done. Also added the same docstrings to the `to_gbq()` function.